tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature

```diff
@@ -257,26 +277,220 @@ Feature: copying file using file id
+    And user "Alice" has created folder "/folder"
+    And user "Alice" has uploaded file with content "some data" to "/folder/test.txt"
+    And we save it into "FILEID"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | folder            |
+      | space           | Personal          |
+      | sharee          | Brian             |
+      | shareType       | user              |
+      | permissionsRole | <permission-role> |
+    And user "Brian" has a share "folder" synced
+    And user "Alice" has created a space "project-space" with the default quota using the Graph API
+    And user "Alice" has sent the following space share invitation:
+      | space           | project-space |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | <space-role>  |
+    When user "Brian" copies a file "Shares/folder/test.txt" into "/" inside space "project-space" using file-id path "<dav-path>"
     Then the HTTP status code should be "201"
-    And for user "Alice" folder "/" of the space "first-project-space" should contain these files:
-      | firstProjectSpacetextfile.txt  |
-      | secondProjectSpacetextfile.txt |
-    And for user "Alice" folder "/" of the space "second-project-space" should contain these files:
-      | secondProjectSpacetextfile.txt |
+    And for user "Brian" folder "folder" of the space "Shares" should contain these files:
+      | test.txt |
+    And for user "Brian" folder "/" of the space "project-space" should contain these files:
+      | test.txt |
+    And for user "Alice" folder "/" of the space "project-space" should contain these files:
+      | test.txt |
     Examples:
-      | dav-path                          |
-      | /remote.php/dav/spaces/<<FILEID>> |
-      | /dav/spaces/<<FILEID>>            |
+      | permission-role | space-role    | dav-path                          |
+      | Viewer          | Manager       | /remote.php/dav/spaces/<<FILEID>> |
+      | Viewer          | Space Editor  | /remote.php/dav/spaces/<<FILEID>> |
+      | Editor          | Manager       | /remote.php/dav/spaces/<<FILEID>> |
+      | Editor          | Space Editor  | /remote.php/dav/spaces/<<FILEID>> |
+      | Uploader        | Manager       | /remote.php/dav/spaces/<<FILEID>> |
+      | Uploader        | Space Editor  | /remote.php/dav/spaces/<<FILEID>> |
+      | Viewer          | Manager       | /dav/spaces/<<FILEID>>            |
+      | Viewer          | Space Editor  | /dav/spaces/<<FILEID>>            |
+      | Editor          | Manager       | /dav/spaces/<<FILEID>>            |
+      | Editor          | Space Editor  | /dav/spaces/<<FILEID>>            |
+      | Uploader        | Manager       | /dav/spaces/<<FILEID>>            |
+      | Uploader        | Space Editor  | /dav/spaces/<<FILEID>>            |
+
+
+  Scenario Outline: sharee tries to copy a file from shares to project space
```

Comment:
duplicate scenario title

Reply:
> Scenario Outline: sharee copies a file from shares to project space 

It is for successful copy scenario 

> Scenario Outline: sharee tries to copy a file from shares to project space

It is for unsuccessful copy scenario due to of read permission.

I tried to make it different using the `tries` word. Should I make any changes?

Reply:
my eyes didn't see tries. :+1: 
it is good.